src/System.Text.RegularExpressions/src/System.Text.RegularExpressions.csproj

```diff
@@ -12,31 +12,38 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
   <ItemGroup>
-    <Compile Include="System\Text\RegularExpressions\HashtableExtensions.cs" />
+    <Compile Include="System\Collections\Generic\ValueListBuilder.Pop.cs" />
```

Comment:
This should probably merge into the rest of VLB.

Reply:
Stephen advised to keep the Pop method out

Reply:
Hmm I believe it was Stephen, or Jan. But that was before we had the Indexer in it. I don't have a strong opinion here...

Reply:
IMHO, Pop as an operation doesn't really belong on the type; it's just how Regex happens to use it.  My original suggestion had actually been (or what I'd intended) for Regex to just have a static helper that took a ValueListBuilder by ref, rather than having it be a partial class instance method like this.